modules/run/artifacts/opt/jboss/container/java/run/run-java.sh

```diff
@@ -217,23 +217,32 @@ get_classpath() {
   echo "${cp_path}"
 }
 
-# Ensure that the running UID has the "jboss" passwd metadata
-# XXX: Maybe we should make this an entrypoint for the image?
-function configure_passwd() {
-  # OPENJDK-533: this file is only writeable if the image uses the
-  # nss_wrapper module. ubi8/openjdk-17 does not.
-  if [ -w "$HOME/passwd" ]; then
-    sed "/^jboss/s/[^:]*/$(id -u)/3" /etc/passwd > "$HOME/passwd"
-  fi
```

Comment:
This is an unrelated change. I'd prefer if we did that separately in case we need to revert.

Reply:
We can, sure.